core/blockchain/stxo.go

```diff
@@ -77,6 +80,8 @@ type SpentTxOut struct {
 	PkScript   []byte // The public key script for the output.
 	BlockHash  hash.Hash
 	IsCoinBase bool // Whether creating tx is a coinbase.
+	TxIndex    uint32
```

Comment:
the new added two field in ‘stxo’ type are not well-documented. the design purpose , why they are designed, how to use them, and better naming are required.